optimize/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -88,8 +88,8 @@ These values control mechanisms of Optimize related security, e.g. security head
 | security.auth.token.secret                       | null            | Optional secret used to sign authentication tokens, it's recommended to use at least a 64-character secret. If set to `null` a random secret will be generated with each startup of Optimize.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-| security.auth.superUserIds                       | [ ]             | List of user IDs that are granted full permission to all collections, reports, and dashboards. <br /><br /> Note: For reports, these users are still required to be granted access to the corresponding process/decision definitions in Camunda 7 Admin. See [Authorization Management](./authorization-management.md).                                                                                                                                                                                                                                                                                                                                                                                      |
-| security.auth.superGroupIds                      | [ ]             | List of group IDs that are granted full permission to all collections, reports, and dashboards. All members of the groups specified will have superuser permissions in Optimize. <br /><br />Note: For reports, these groups are still required to be granted access to the corresponding process/decision definitions in Camunda 7 Admin. See [Authorization Management](./authorization-management.md).                                                                                                                                                                                                                                                                                                    |
```

Comment:
@RomanJRW, I don't know what to do with these links if we delete that page.

Reply:
We can delete these two lines, they are C7-only config anyway 👍